cogs/dice_roller.py

```diff
@@ -3,38 +3,30 @@
 import random
 
 
-class Example(commands.Cog):
+class DiceRoller(commands.Cog):
     def __init__(self, client):
         self.client = client
 
     @commands.Cog.listener()
     async def on_ready(self):
-        print('Example cog is loaded')
+        print('Dice rolling cog is loaded')
 
     @commands.command()
     async def roll(self, ctx, dice: str):
-
         cleaned_dice = dice.lower()
-
         dice_list = dice.split('d')
-
         dice_number = int(dice_list[0])
-
         dice_type = int(dice_list[1])
-
         if dice_number == 1:
-
             dice_total = random.randint(1, dice_type)
-
             await ctx.send(
                 f'*{ctx.message.author} rolls {dice} for {dice_total}*'
```

Comment:
Demo the dice rolling please.

Reply:
![image](https://user-images.githubusercontent.com/9323485/59569251-e0c92d00-9043-11e9-9a20-21fadc7c8dbc.png)